autotelic/fastify-csv-import
# feat: fastify v3 support

# Summary

- Downgraded multipart to 6.0.0 to support fastify v3

# Test Plan

- Install dependencies with npm
- Make sure app still works : run example app with node:
```sh
node ./example/index.js
```
- In a separate terminal, make the following requests:
```sh
http -f POST :3000/csv/import file@example/fixtures/valid-products.csv # should respond with an array of products
http -f POST :3000/csv/import file@example/fixtures/large-file.csv # should return an error with appropriate message 
http -f POST :3000/csv/import file@example/fixtures/invalid-fixed-price.csv # should return an error with appropriate message 
```
